### PR TITLE
Add survey banner

### PR DESCRIPTION
**Pull request by BenWilson2**

### CODE REVIEW

**Comment by BenWilson2** [comment]
Screenshot (in case you can't get StackBlitz to render) 
Screenshot 2024-12-19 at 8 55 22 PM

File: website/docusaurus.config.ts [hunk 1]
```diff
@@ -218,6 +218,14 @@ const config: Config = {
       theme: prismThemes.vsDark,
       darkTheme: prismThemes.vsDark,
     },
+    announcementBar: {
+      id: "survey_bar",
+      content:
+        'Help us improve MLflow by taking our <a target="_blank" rel="noopener noreferrer" href="https://surveys.training.databricks.com/jfe/form/SV_cA2jrfBjs6vi6SG">survey</a>!',
+      backgroundColor: "#72A1ED",
+      textColor: "#091E42",
```
Comment (daniellok-db) [inline website/docusaurus.config.ts:226]:
```suggestion
      backgroundColor: "#0194e2",
      textColor: "#ffffff",
```

Screenshot 2024-12-20 at 10 01 00 AM

**Review by daniellok-db — APPROVED** [review]
overall lgtm!

File: website/docusaurus.config.ts (hunk 1 above)
Comment (BenWilson2) [reply website/docusaurus.config.ts:226]: A most excellent change :D

**Pull request merged**